RecoTauTag/Configuration/python/HPSPFTaus_reminiAOD_cff.py

```diff
@@ -0,0 +1,1354 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
if I understand correctly, this file has only one single application as a target (the upcoming 76X reminiaod).
Otherwise, this can't possibly be maintained in line with all mainstream tau reco configuration.

Is there really no way to just use the already setup structure to modify tau sequence in the fragment used by PAT: updateHPSPFTaus_cff ?